README.md

```diff
@@ -156,9 +159,10 @@ Inspired by the [awesome](https://github.com/sindresorhus/awesome) list thing. F
  - [MongoDBLink](https://github.com/zbjornson/MongoDBLink) - Community Mathematica driver
 
 ### Perl
- - [mongo-perl-driver](https://github.com/mongodb-labs/mongo-perl-driver) - Official Perl driver
+ - [mongo-perl-driver](https://github.com/mongodb-labs/mongo-perl-driver) - Official MongoDB Perl driver (no longer nmaintained)
```

Comment:
If it's no longer maintained then the awesomeness is relative, we should probably just remove it :(